tests/test_console.py

```diff
@@ -74,13 +77,18 @@ def test_send_data_no_endpoint(self):
 Postgresql = testing.postgresql.PostgresqlFactory(cache_initialized_db=True)
 
 
+def get_test_data():
+    SITE_ROOT = os.path.realpath(os.path.dirname(__file__))
+    json_url = os.path.join(SITE_ROOT, 'test_data', 'test_response_1.json')
+    responses_json = []
+    with open(json_url) as json_data:
```

Comment:
This doesn't look quite right to me - if the json is already a json array, it should be contained in a list when converted by `json`. If not could you just call `list(json_data)` on it?